pyro/contrib/forecast/forecaster.py

```diff
@@ -182,14 +196,17 @@ def __init__(self, model, data, covariates, *,
                  learning_rate_decay=0.1,
                  num_steps=1001,
                  log_every=100,
-                 init_scale=0.1):
+                 init_scale=0.1,
+                 num_particles=1,
```

Comment:
do we need docstrings for num_particles, learning_rate_decay, etc.?

Reply:
Do you mind if I add those in a future PR to avoid blocking follow-up PRs?